frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/_add-capacity-modal.scss

```diff
@@ -19,18 +23,27 @@
       left: 6.6rem;
       top: 8.5rem;
     }
-  }
-  
+  } 
   padding-top: 2em;
 }
 
 .add-capacity-modal__span {
-  margin-left: 3em;
-  color: $pf-color-black-500;
-  button {
-    position: absolute;
-    left: 9.1rem;
-    top: 3.8rem
-  }
+  margin-left: 3.5rem;
+}
+
+.add-capacity-modal__input--width {
+  max-width: 250px !important;
```

Comment:
@a2batic Why there is a need to add important here. Instead of that you can make the selector specific. 
Its considered as an anti-pattern. https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
ack